### PR TITLE
updating set-acl docs per PR#901

### PR DESCRIPTION
**Pull request by sdwheeler**
PR #901 is very old. The changes cannot be merged easily without conflicts. Updated the Set-ACL docs per PR #901 so that it can be closed without merging. 

Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-fixPR901/reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-fixPR901/reference/4.0/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-fixPR901/reference/5.0/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-fixPR901/reference/5.1/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-fixPR901/reference/6/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201711072349220812-1844%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=rCz3HVBoE2W0UotSw7QPBNAmC4zu9P7vjRXzSh3DFkY%3D&st=2017-11-08T00%3A25%3A20Z&se=2017-12-09T00%3A30%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sdwheeler** [comment]
@zjalexander Please review and merge.

File: reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md [hunk 1]
```diff
@@ -9,101 +9,129 @@ title:  Set-Acl
...
+C:\Temp directory and all of its subdirectories.
 
-The first command gets the security descriptor of the File0.txt file in the current directory and uses the assignment operator (=) to store it in the $NewACL variable.
+The first command gets the security descriptor of the File0.txt file in the current directory and
+uses the assignment operator (=) to store it in the $NewACL variable.
 
-The first command in the pipeline uses the Get-ChildItem cmdlet to get all of the text files in the C:\Temp directory.
-The **Recurse** parameter extends the command to all subdirectories of C:\temp.
+The first command in the pipeline uses the Get-ChildItem cmdlet to get all of the text files in the
+C:\Temp directory. The **Recurse** parameter extends the command to all subdirectories of C:\temp.
```
Comment (zjalexander) [inline reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md]: When editing files, can you please add semantic linebreaks? http://rhodesmill.org/brandon/2012/one-sentence-per-line/ basically a linebreak after the end of each sentence. This makes the diffs a lot easier when we read them in the future

Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md]: @zjalexander Will do. My editor is configured to break lines at 100 columns.

Comment (joeyaiello) [reply reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md]: Yes, please. I didn't get semantic line breaks until I started doing some longer form diffs/reviews. Makes life so much easier :)

Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md]: Do you want me to re-edit these?

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-fixPR901/reference/3.0/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-fixPR901/reference/4.0/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-fixPR901/reference/5.0/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-fixPR901/reference/5.1/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Security/Set-Acl.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-fixPR901/reference/6/Microsoft.PowerShell.Security/Set-Acl.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C10%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201711101911266319-1844%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=mcmPhJypWd1QmMWaKlpgPmpeZwDZgAATp95W570d%2BTc%3D&st=2017-11-10T19%3A37%3A18Z&se=2017-12-11T19%3A42%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**